src/bundle/Core/Command/ReindexCommand.php

```diff
@@ -469,4 +477,28 @@ private function getNumberOfCPUCores(): int
 
         return $cores;
     }
+
+    /**
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\InvalidArgumentException
+     */
+    private function getSearchEngineAlias(): string
+    {
+        if ($this->searchIndexer instanceof IncrementalIndexer) {
+            return $this->searchIndexer->getName();
+        }
+
+        $repositoryConfig = $this->repositoryConfigurationProvider->getRepositoryConfig();
+        $searchEngineAlias = $repositoryConfig['search']['engine'] ?? null;
+
+        if (null === $searchEngineAlias) {
+            throw new RuntimeException(
+                sprintf(
+                    '"%s" repository has no search engine configured',
+                    $this->repositoryConfigurationProvider->getCurrentRepositoryAlias()
+                )
+            );
+        }
+
+        return ucfirst($searchEngineAlias);
```

Comment:
Would prefer not to uppercase. While it's "nicer", it also prevents checking for this particular string in configuration, when someone is looking for it.
```suggestion
        return $searchEngineAlias;
```

Reply:
https://github.com/ibexa/core/pull/607/commits/71d64340bf14dc47f30239046426fab1bd07b679